### DISCUSSION TITLE
Is there a way to set frontmatter property for every list in the project?

### DISCUSSION THREAD

**Discussion by MikiVanousek** [jpanther/congo#121] [Q&A]
I want to have my posts separated into several categories, each post belonging to one and only one category. I reconned the best way to do this is to create subfolders and list in each sub folder.

So far I made only one category `foss`. This is my content folder structure:
```
content
├── horolux
│   ├── horolux-scs.png
│   ├── index.md
│   └── wake-up.jpeg
├── _index.md
└── posts
    ├── foss
    │   ├── _index.md
    │   └── root-arrow.md
    └── _index.md
```

This means, that  when I open the posts list, I see a list of category lists. Each category is a list of posts. 

I want each post to display the date it was made and updated.
![image](https://user-images.githubusercontent.com/33402194/153823964-76baa333-9e85-4581-b66e-c8d235ca9ac4.png)

But, if the list item is not a blog post, but a category, I want the dates to be hidden.
![image](https://user-images.githubusercontent.com/33402194/153824091-e087b93a-12d8-4001-8957-a7b85e3adcd7.png)

I now set this in the category front matter. 
```
---
showDate: false
showDateUpdated: false
title: "Free And Open Source Software"
description: "Everything I wrote about open source"
summary: "Everything I wrote about open source"
---
```
But this means I need to set it for each new category (and potentially subcategory in the future) I create.  I tried setting these in the `[list]` config, but it did not take effect. Is there a way to change this globally? If you are not a post, but a list, do not show your date!

**Comment by jpanther** [comment]
Thanks for the question. I may not be fully understanding your end objective here but a useful feature of Hugo that may help is the special `cascade` front matter parameter. When set, any children of this page will inherit the parameters that are cascaded.

You can essentially do something like this in your `_index.md` front matter to hide the dates on any sub-pages automatically:

```yaml
---
title: "Categories"
cascade:
  showDate: false
  showDateUpdated: false
---
```

The example site in the theme uses this method in [content/samples/_index.md](https://github.com/jpanther/congo/blob/dev/exampleSite/content/samples/_index.md) if you want to see a working example.

**Reply by MikiVanousek** [reply]
Is there a way to apply one set of values to all the articles and another to all the lists? I tried:
```yaml
---
cascade:
- target:
    kind: article
  showDate: true
  showDateUpdated: true
- target:
  kind: list
  showDate: false
  showDateUpdated: false
---
```
But this applies the first set of values to all sub-files.
